### ISSUE TITLE
`WriteConnectionSettingComments()` is not controlled with `includeConnectionSettingComments`  in v2.37.0?

### ISSUE THREAD

**Issue by yyjdelete** [ReversePOCO/EntityFramework-Reverse-POCO-Code-First-Generator#447]
See d955e813.
Before this commit, `IsAzure(bool includeConnectionSettingComments)` use `includeConnectionSettingComments` to see if it will write `Database Edition` and the others, but the logic is lost in this commit.

**Comment by sjh37** [comment]
The logic got moved into a separate function: `WriteConnectionSettingComments()`

There are basically 2 separate parts:

**Part 1**
```c#
// The following connection settings were used to generate this file:
// The following connection settings were used to generate this file:
//     Configuration file:     "EntityFramework.Reverse.POCO.Generator\App.config"
//     Connection String Name: "MyDbContext"
//     Connection String:      "Data Source=(local);Initial Catalog=northwind;Integrated Security=True;Application Name=EntityFramework Reverse POCO Generator"
// ------------------------------------------------------------------------------------------------
```
**Part 2**
```c#
// Database Edition        : Developer Edition (64-bit)
// Database Engine Edition : Enterprise
// Database Version        : 13.0.4001.0
```

Part 1 is controlled via `Settings.IncludeConnectionSettingComments`
Part 2 is now always written.

Thinking about it, I think Part 2 should also be controlled via `Settings.IncludeConnectionSettingComments`

I will make the change.
Thanks for pointing this out.

**Comment by sjh37** [comment]
Fixed, this will be in the next release.

**Comment by sjh37** [comment]
Released in v2.37.1
